fe/src/main/java/org/apache/doris/load/PullLoadSourceInfo.java

```diff
@@ -1,119 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This class has been consistent in the old version. Maybe you should keep it .

Reply:
It ok, I check it, it does no use at all now. and the old load job are gone before Doris version 0.11.